python/lib/dcos/dcos/marathon.py

```diff
@@ -463,22 +463,23 @@ def get_deployment(self, deployment_id):
 
         return deployment
 
-    def get_deployments(self, app_id=None):
-        """Returns a list of deployments, optionally limited to an app.
+    def get_deployments(self, id_filter=None):
```

Comment:
This renaming of the parameter didn't break any tests?

Reply:
This PR didn't trigger the test suite, we'll be able to see this once it's opened against https://github.com/dcos/dcos-core-cli.

But marathon tests are currently broken for a few days, @timcharper has a fix in progress for this (https://github.com/mesosphere/marathon/pull/6428).